tests/DCPS/HelloWorld/publisher.cpp

```diff
@@ -0,0 +1,63 @@
+// -*- C++ -*-
+#include "HelloWorldTypeSupportImpl.h"
+#include <dds/DCPS/Service_Participant.h>
+#include <dds/DCPS/Marked_Default_Qos.h>
+#include "dds/DCPS/StaticIncludes.h"
+#ifdef ACE_AS_STATIC_LIBS
+#include <dds/DCPS/RTPS/RtpsDiscovery.h>
+#include <dds/DCPS/transport/rtps_udp/RtpsUdp.h>
+#endif
+
+#include <tests/Utils/DistributedConditionSet.h>
+
+int ACE_TMAIN(int argc, ACE_TCHAR *argv[])
+{
+  // Production apps should check the return values.
+  // For tests, it just makes the code noisy.
+
+  // Coordination across processes.
+  DistributedConditionSet_rch distributed_condition_set =
+    OpenDDS::DCPS::make_rch<FileBasedDistributedConditionSet>();
+
+  DDS::DomainParticipantFactory_var domain_participant_factory = TheParticipantFactoryWithArgs(argc, argv);
+  DDS::DomainParticipant_var participant =
+    domain_participant_factory->create_participant(HelloWorld::HELLO_WORLD_DOMAIN,
+                                                   PARTICIPANT_QOS_DEFAULT,
+                                                   0,
+                                                   ::OpenDDS::DCPS::DEFAULT_STATUS_MASK);
```

Comment:
To simplify code and reduce noise, consider replacing all cases of DEFAULT_STATUS_MASK with 0 when there is no listener